v-next/hardhat/src/internal/builtin-plugins/network-manager/edr/stack-traces/stack-trace-solidity-errors.ts

```diff
@@ -420,7 +420,41 @@ class SolidityCallSite implements NodeJS.CallSite {
     return 0;
   }
 
+  // Extracted and adapted from source-map-support package, which extracts it from V8
   public toString(): string {
-    return "[SolidityCallSite]";
+    let fileLocation = this.getFileName();
+
+    if (this.getLineNumber() !== null) {
+      fileLocation += `:${this.getLineNumber()}`;
+    }
+
+    let line = "";
+    const functionName = this.getFunctionName();
+    let addSuffix = true;
+    const isConstructor = this.isConstructor();
+    const isMethodCall = !(this.isToplevel() || isConstructor);
```

Comment:
I've checked the definition of `isToplevel()`, and it always seems to return `false`. Am I missing something?

Reply:
Some of these CallSite interface functions are returning fixed values, but probably that's temporary. Since I'm not really sure, I didn't exclude the logic from the toString() function, in case in the future those functions may be of use